test/integration/lib/philipsHue_server.js

```diff
@@ -0,0 +1,83 @@
+var express = require ('express');
+var bodyParser = require('body-parser');
+var path = require('path');
+
+var philipsHue_server = (function() {
+
+	var welcome_page = '/html/philips_initial.html';		// Philips Hub webpage
```

Comment:
Nit: This string is only [used once](https://github.com/fxbox/foxbox/pull/176/files#diff-aff726e949c461077c3a77d720cc4ad1R38). If we put the string directly there, people will have enough context to understand it's a path to the welcome page. Hence, I'd remove this var.